### PR TITLE
【Abondoned for commit lint problem 】:  implement event watch feature

### PR DESCRIPTION
**Pull request by zhouhaoA1**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
implement event watch feature and improve get/list performance
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/synchromanager/clustersynchro/informer/resourceversion_informer.go [hunk 1]
```diff
@@ -79,15 +79,15 @@ func (informer *resourceVersionInformer) HandleDeltas(deltas cache.Deltas, isInI
 				}
 
 				informer.handler.OnAdd(d.Object, isInInitialList)
-				break
+				continue
```
Comment (xyz2277) [inline pkg/synchromanager/clustersynchro/informer/resourceversion_informer.go:82]: break is ok, continue is not needed

File: pkg/synchromanager/clustersynchro/resource_synchro.go [hunk 2]
```diff
@@ -361,12 +350,31 @@ func (synchro *ResourceSynchro) handleResourceEvent(event *queue.Event) {
 
 	obj, ok := event.Object.(runtime.Object)
 	if !ok {
-		return
+		if _, ok = event.Object.(cache.DeletedFinalStateUnknown); !ok {
+			return
+		} else {
```
Comment (xyz2277) [inline pkg/synchromanager/clustersynchro/resource_synchro.go:355]: else here is not needed

File: pkg/synchromanager/clustersynchro/informer/resourceversion_informer.go (hunk 1 above)
Comment (zhouhaoA1) [reply pkg/synchromanager/clustersynchro/informer/resourceversion_informer.go:82]: ok

File: pkg/synchromanager/clustersynchro/resource_synchro.go (hunk 2 above)
Comment (zhouhaoA1) [reply pkg/synchromanager/clustersynchro/resource_synchro.go:355]: ok

**Pull request closed without merging**